src/Mapbender/CoreBundle/Resources/views/Element/search_router.html.twig

```diff
@@ -8,7 +8,7 @@
         {% endfor %}
     </div>
     <div class="search-action-buttons">
-        <span data-action="reset" class="btn btn-sm btn-default">{{ 'mb.actions.reset' | trans }}</span>
-        <span data-action="search" class="btn btn-sm btn-default">{{ 'mb.actions.search' | trans }}</span>
+        <button data-action="reset" class="btn btn-sm btn-default">{{ 'mb.actions.reset' | trans }}</button>
```

Comment:
Does this work as expected?
HTML button with no `type` attribute defaults to type="submit" and will implicitly submit the enclosing form. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).

Can you please verify that
* Clicking on the search button only executes the search once
* Clicking on the reset button does not execute a search

Reply:
hi @werrolf,
I checked out the branch and followed your instruction (with F12 network opened). Tested with Firefox

-     Clicking on the search button only executes the search once -> yes
-     Clicking on the reset button does not execute a search -> yes

So it looks it works fine